modules/hcp-ibm-power-create-heterogeneous-nodepools-agent-hc.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ibm-power-create-heterogeneous-nodepools-agent-hc_{context}"]
+= Creating heterogeneous node pools on agent hosted clusters
+
+include::hcp-ibm-power-create-heterogeneous-nodepools-agent-hc-con.adoc[leveloffset=+2]
```

Comment:
```suggestion

```
I just realized that you were including a module within another module. I think this might one of the reasons why the tests are failing.

Reply:
Ah. I see why the preview looks odd. See https://github.com/openshift/openshift-docs/pull/92359/files#r2146094888